build-farm/jdk10u/Linux/x64/hotspot/setup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+################################################################################
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+################################################################################
+#
+# Setup for JDK 10 builds on AdoptOpenJDK build farm nodes
+#
+################################################################################
+
+[ -r /opt/rh/devtoolset-2/root/usr/bin/gcc ] && export CC=/opt/rh/devtoolset-2/root/usr/bin/gcc
+[ -r /opt/rh/devtoolset-2/root/usr/bin/g++ ] && export CXX=/opt/rh/devtoolset-2/root/usr/bin/g++
```

Comment:
do we need this in separate files? this is a universal setup command for all linux x64 builds. It's also the same as `build-farm/jdk10u/Linux/x64/openj9/setup.sh`.

Reply:
Yeah I was not a fan of the duplication. My issue was I did not feel it was clear what configuration would be built with when it was unified. I have reverted it to a single file which can be seen at: https://github.com/johnoliver/openjdk-build/blob/overhaul-build-scripts-2/build-farm/set-platform-specific-configurations.sh will get @karianna to merge that version into this PR

Reply:
Have now fixed this to remove the duplication so there is a file per OS, it looks fairly reasonable